alvr/session/src/settings.rs

```diff
@@ -865,6 +864,15 @@ pub struct HapticsConfig {
     pub min_duration_s: f32,
 }
 
+#[derive(SettingsSchema, Serialize, Deserialize, Clone)]
+pub struct HandSkeletonConfig {
+    #[schema(flag = "real-time")]
```

Comment:
This works by pulling tracking from the separate trackers, right? If so I think this should have a note that real-time switching doesn't work for games that you might want to disable it for because it's buggy

Reply:
I guess it's up for testing, but we hope it will work

Reply:
I really want a small note like "Real-time switching only disables tracking, removing separate controllers requires SteamVR restart"

Reply:
It's somewhat explained in the preset